adding_species/contributing.ipynb

```diff
@@ -419,32 +406,69 @@
     "```\n",
     "and if all good we are ready to commit with\n",
     "```\n",
+    "git add stdpopsim/catalog/AnoGam/*.py\n",
     "git commit -am \"added mosquito genome woot\"\n",
+    "\n",
     "```\n",
+    "depending on how well you follow official style, the pre-commit hooks \n",
+    "that call the auto-formatting tools `black` and `flake8` \n",
+    "may or may not reformat your code. if they do simply rerun `git commit`\n",
+    "and you should now successfully be able to commit your code. \n",
```

Comment:
Is it always that simple? eg. Gregor's Bos taurus situation https://github.com/popsim-consortium/stdpopsim/pull/609

Reply:
gosh i haven't had black + flake8 fail for me yet, but i bet it can

Reply:
Flake8 won't reformat stuff, it'll just complain. That's another thing worth leaving an error in and showing how to fix.